backend/gatekeeper/views.py

```diff
@@ -71,6 +76,14 @@ def delete(self, request):
         self.sts_arguments['params'] = {k: v for k, v in request.query_params.items() if k.startswith('$')}
         return self.handle_request(request)
 
+    def options(self, request, *args, **kwargs):
+        self.sts_arguments['params'] = {k: v for k, v in request.query_params.items() if k.startswith('$')}
```

Comment:
Now that this is in three places, could it be a method "get STA parameters" or something?

Reply:
Sure, that is a good point!

Reply:
I'm wondering, maybe these should always be set. Any query parameter should always be passed to the STS right? No matter the HTTP method, be it POST or GET for instance.

Reply:
Done